config/jobs/kubernetes/sig-scalability/sig-scalability-cleanup.yaml

```diff
@@ -31,3 +31,49 @@ periodics:
       - ci-kubernetes-e2e-gci-gce-scalability
       # Delete snapshots older than x days
       - "30"
+
+- name: ci-kubernetes-scalability-cleanup-golang-builds-canary
+  cluster: k8s-infra-prow-build
+  interval: 24h
+  labels:
+    preset-service-account: "true"
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+  - org: kubernetes
+    repo: perf-tests
+    base_ref: master
+    path_alias: k8s.io/perf-tests
+  annotations:
+    # TODO(ameukam): update with email alert
+    testgrid-dashboards: sig-k8s-infra-canaries
+    testgrid-tab-name: golang-build-5k-project-snapshots-cleanup-canary
+    testgrid-alert-email: ameukam@gmail.com
+    testgrid-num-failures-to-alert: '3'
+  spec:
+    serviceAccountName: boskos-janitor
```

Comment:
@spiffxp Is it ok to use this job or do I need to create a separate SA with custom role ?

Reply:
The default service account should also have rights to do this

https://github.com/kubernetes/k8s.io/blob/main/infra/gcp/bash/prow/ensure-e2e-projects.sh#L71-L83

Reply:
If you add a TODO that we should revisit this that's good enough for now for me